spec/javascripts/mobile/mobile_comments_spec.js

```diff
@@ -155,6 +155,18 @@ describe("Diaspora.Mobile.Comments", function(){
       expect(this.toggleReactionsLink.text().trim()).toBe("1 comment");
       expect(this.toggleReactionsLink.attr("href")).toBe("/posts/" + postGuid + "/comments.mobile");
     });
+
+    it("Creates the reaction link when there are no reactions (french locale)", function() {
+      var parent = this.toggleReactionsLink.parent();
+      var postGuid = this.bottomBar.parents(".stream-element").data("guid");
+      this.toggleReactionsLink.remove();
+      parent.prepend($("<span/>", {"class": "show-comments"}).text("Aucun commentaire"));
+
+      Diaspora.Mobile.Comments.increaseReactionCount(this.bottomBar);
+      this.toggleReactionsLink = this.bottomBar.find(".show-comments").first();
+      expect(this.toggleReactionsLink.text().trim()).toBe("1 comment");
```

Comment:
Can't we preload the number (with gon?) instead of parsing the text representation?